src/lib/pmd/CustomRuleRegistrar.ts

```diff
@@ -0,0 +1,83 @@
+import fs = require('fs');
+import path = require('path');
+import util = require('util');
+
+export const CUSTOM_RULE_REGISTER = path.join('catalogs', '.custom_paths.json');
+
+/**
+ * Handles registering custom rules into CustomRuleRegister JSON
+ * 
+ * TODO: verify validity of path. Better to fail now than later while creating catalog.
+ */
+
+export class CustomRuleRegistrar {
+
+    public async createEntries(language: string, location: string[]) {
+
+        // Fetch current entries in custom rule register as a Map
+        var currentEntries = await this.readCurrentEntries();
```

Comment:
Never use `var`. Always use `const` if you won't need to reassign the variable, or `let` if you will have to reassign it.